developer/src/kmc-keyboard-info/test/fixtures/khmer_angkor/build/.gitattributes

```diff
@@ -0,0 +1 @@
+khmer_angkor.js -text
```

Comment:
... looks like that wasn't enough to have it treated as binary for the review.  Github definitely reports it as a 5400+ line added file.

Reply:
This was to stop CRLF vs LF manipulation that broke file sizes on different platforms -- needed for the metadata test to pass.